src/app/collection/collection.component.html

```diff
@@ -0,0 +1,48 @@
+<div *ngIf="_loading || _loadingMetadata">
+  <mat-progress-bar mode="buffer"></mat-progress-bar>
+</div>
+
+<h2>Information about all Multimedia Objects in the database</h2>
+<h3>Click on an object to see all metadata</h3>
+<mat-paginator [length]="length"
+               [pageSize]="pageSize"
+               [pageSizeOptions]="pageSizeOptions"
+               (page)="pagination($event)">
+</mat-paginator>
+<table mat-table [dataSource]="data" multiTemplateDataRows>
```

Comment:
I highly suggest the use of `[trackBy]=trackById` with a method 

```
trackById(idx: number, item: T){
  return item.id;
}
```

with `T` being the type of `data` (and it actually having a unique identifier like `.id`).
This results in a much more conservative re-rendering of the table (and thus, in a performance increase) and is [generally considered to be good practice](https://material.angular.io/cdk/table/overview#trackby)